### PR TITLE
fixes #470 replaces FluoFileOutputFormat with Key Value generator

### LINKED ISSUE #470: Consider replacing FluoFileOutputFormat with an Accumulo Key Builder
See discussion on ngageoint/geowave#157

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java [hunk 1]
```diff
@@ -28,12 +29,12 @@
 import org.apache.hadoop.io.Text;
 
 /**
- * This class allows building Accumulo mutations that are in the Fluo data format. This class is
- * intended to be used with {@link AccumuloOutputFormat} inorder to seed an initialized Fluo table
- * on which no transactions have executed.
+ * This class allows generates Accumulo mutations that are in the Fluo data format. This class is
```
Comment (mikewalch) [inline modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java]: could be `This class generates Accumulo`

File: modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java [hunk 2]
```diff
@@ -28,12 +29,12 @@
...
+ * intended to be used with {@link AccumuloOutputFormat} or (@link {@link BatchWriter} inorder to
```
Comment (mikewalch) [inline modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java]: Should there two links?

File: modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java (hunk 1 above)
Comment (keith-turner) [inline modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java]: c9e31cc

File: modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java (hunk 2 above)
Comment (keith-turner) [inline modules/mapreduce/src/main/java/io/fluo/mapreduce/FluoMutationGenerator.java]: c9e31cc

**Comment by mikewalch** [comment]
+1

**Pull request merged**